tools/topology/topology2/development/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+# Array of "input-file-name;output-file-name;comma separated pre-processor variables"
+set(TPLGS
+# CAVS SSP topology for MTL for electrical validation
+# Default MCLK = 24576000, BCLK = 3072000, FSYNC = 48000,
+# TDM_SLOTS = 2, TDM_WIDTH = 32
+# Verify with
+# MCLK = 38400000, BCLK = 4800000, FSYNC = 48000,
+# TDM_SLOTS = 4, TDM_WIDTH = 25
+"cavs-nocodec-val\;sof-mtl-nocodec-1\;PLATFORM=mtl,NUM_DMICS=2,\
+MCLK=38400000,BCLK=4800000,TDM_SLOTS=4,TDM_WIDTH=25,\
+PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-ace-mtl-nocodec-1.bin"
```

Comment:
the .bin is only needed for nhlt blob debugging, so is it not needed if you don't want to pollute the build dir. But if you really need it, it is fine.

Reply:
removed multiple bin generation